### PR TITLE
update esignet-default.properties for MOSIP VCI download

### PR DESCRIPTION
**Pull request by vijay151096**

### CODE REVIEW

File: esignet-default.properties [hunk 1]
```diff
@@ -480,7 +480,7 @@ mosip.esignet.vci.key-values={\
           },\
    'latest' : {\
               'credential_issuer': '${mosip.esignet.vci.identifier}', 	\
-              'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/vci/credential', \
+              'credential_endpoint': '${mosip.esignet.host}${server.servlet.path}/vci/credential', \
```
Comment (ghost) [inline esignet-default.properties]: Pls change this to ${mosip.esignet.domain.url}. kinldy check the below line https://github.com/mosip/mosip-config/blob/collab1/esignet-default.properties#L269

File: esignet-default.properties [hunk 2]
```diff
@@ -444,7 +444,7 @@ mosip.esignet.vci.supported.jwt-proof-alg={'RS256','PS256'}
 mosip.esignet.vci.key-values={\
   'v11' : { \
               'credential_issuer': '${mosip.esignet.vci.identifier}', 	\
-              'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/vci/credential', \
+              'credential_endpoint': '${mosip.esignet.host}${server.servlet.path}/vci/credential', \
```
Comment (ghost) [inline esignet-default.properties]: Pls change this to ${mosip.esignet.domain.url}. kinldy check the below line https://github.com/mosip/mosip-config/blob/collab1/esignet-default.properties#L269

File: esignet-default.properties (hunk 1 above)
Comment (vijay151096) [reply esignet-default.properties]: URL Modified.

File: esignet-default.properties (hunk 2 above)
Comment (vijay151096) [reply esignet-default.properties]: URL Modified.

**Review by ghost — APPROVED** [review]

**Pull request merged**